octoprint_mrbeam/templates/conversion_dialog.jinja2

```diff
@@ -398,10 +398,10 @@
                                                     id="parameter_assignment_engraving_mode_precise_btn"
                                                     data-bind="event: {click: updateEngravingMode}"
                                                     class="btn btn-default active">{{ _('Recommended') }}</button>
-                                            <button type="button" value="fast"
+                                            {#<button type="button" value="fast"
```

Comment:
if we are commenting it we should add a todo  linking to a ticket where we will solve this

Reply:
Done